src/pin-code/pin-code.js

```diff
@@ -23,6 +23,7 @@ export default class PinCode extends React.Component<PropsT, StateT> {
 
   _inputRefs = new MultiRef<number, HTMLInputElement>();
 
+  hasMask = typeof this.props.mask === 'boolean' ? this.props.mask : false;
```

Comment:
Not sure we need to store this if only used once - the name is a bit misleading. if `this.props.mask` is a string i would expect `hasMask` to be true

Reply:
Yeah, I will delete it.